src/server/db_slice.cc

```diff
@@ -749,11 +751,11 @@ void DbSlice::FlushDbIndexes(const std::vector<DbIndex>& indexes) {
     ClearOffloadedEntries(indexes, db_arr_);
 
   DbTableArray flush_db_arr(db_arr_.size());
+
   for (DbIndex index : indexes) {
-    auto& db = db_arr_[index];
-    CHECK(db);
```

Comment:
Any reason to remove the variable and the `CHECK()`?

Reply:
just because in this block we move it, and then recreate it, I thought it's confusing.